src/App.js

```diff
@@ -1,5 +1,94 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    const carNameString = await Console.readLineAsync(
+      "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+    );
+    let carNames = carNameString
+      .split(",")
+      .map((carName) => this.validateName(carName));
+    carNames = this.validateDuplicateName(carNames);
+
+    let playNum = await Console.readLineAsync("시도할 횟수는 몇 회인가요?\n");
+    playNum = this.isPositiveNumber(playNum);
+
+    const racingCars = carNames.map((carName) => {
+      return { name: carName, score: 0 };
+    });
+
+    Console.print("\n실행 결과");
+    for (let i = 0; i < playNum; i++) {
+      racingCars.map((car) => {
+        this.randomForward(car);
+        this.printRacingcarInfo(car);
+      });
+      Console.print("\n");
+    }
+
+    Console.print(
+      `최종 우승자 : ${this.getWinner(racingCars)
+        .map((car) => car.name)
+        .join(", ")}`
+    );
+  }
+
+  randomForward(car) {
+    if (MissionUtils.Random.pickNumberInRange(0, 9) >= 4) {
+      car.score += 1;
+    }
+  }
+
+  printRacingcarInfo(car) {
+    Console.print(`${car.name} : ${"-".repeat(car.score)}`);
+  }
+
+  getWinner(racingCars) {
+    let winner = [];
+
+    racingCars.map((car) => {
+      if (winner.length === 0) {
+        winner.push(car);
+      } else if (winner[0].score === car.score) {
+        winner.push(car);
+      } else if (winner[0].score < car.score) {
+        winner = [car];
+      }
+    });
+
+    return winner;
+  }
```

Comment:
else와 else if 를 가독성 측면에서 지양한다고 해서 아래와 같은 방법은 어떤가요? 
```suggestion
  getWinner(racingCars) {
    let winner = [];

    racingCars.map((car) => {
      if (winner.length === 0) {
        winner.push(car);
      }
       if (winner[0].score === car.score) {
        winner.push(car);
      }
      if (winner[0].score < car.score) {
        winner = [car];
      }
    });

    return winner;
  }
```
```suggestion

  getWinner(racingCars) {
      const maxScore = Math.max(...racingCars.map(car => car.score));
      return racingCars.filter(car => car.score === maxScore);
  }
```

Reply:
조언 감사합니다!
else가 없을 경우엔 두 조건문 모두에 해당하는 경우 둘 다 실행되어버리는 것을 방지하기 위해 명시적으로 else를 써주는 편이었는데,

else를 없애고 각 조건들을 서로 독립적으로 유지하는 것이 더 좋아 보이네요!